src/chains/const/index.ts

```diff
@@ -0,0 +1,51 @@
+import { AdditionalTargetChainId, ChainInfo, SupportedChainId } from '../types'
+
+import { mainnet } from '../details/mainnet'
+import { gnosisChain } from '../details/gnosis'
+import { arbitrumOne } from '../details/arbitrum'
+import { base } from '../details/base'
+import { sepolia } from '../details/sepolia'
+
+import { optimism } from '../details/optimism'
+import { polygon } from '../details/polygon'
+
+/**
+ * Details of all supported chains.
+ */
+export const ALL_SUPPORTED_CHAINS_MAP: Record<SupportedChainId, ChainInfo> = {
+  [SupportedChainId.MAINNET]: mainnet,
+  [SupportedChainId.GNOSIS_CHAIN]: gnosisChain,
+  [SupportedChainId.ARBITRUM_ONE]: arbitrumOne,
+  [SupportedChainId.BASE]: base,
+  [SupportedChainId.SEPOLIA]: sepolia,
+}
+
+/**
+ * All supported chains.
+ */
+export const ALL_SUPPORTED_CHAINS = Object.values(ALL_SUPPORTED_CHAINS_MAP)
```

Comment:
Unfortunately typescript doesn't derive type for that, we should specify it explicitly here

Reply:
what do you mean? It does to me 
<img width="664" alt="image" src="https://github.com/user-attachments/assets/7503f8d1-b3b4-45ac-919b-2878eae85b42" />

Reply:
Wow, Idea is more stupid that VSCode.
All good then